Makefile

```diff
@@ -12,4 +12,8 @@ lint: fmt
 
 .PHONY: fmt
 fmt:
-	bash -c "diff -u <(echo -n) <(gofmt -d -s .)"
+	gofmt -w -s .
+
+.PHONY: generate
+generate:
+	openapi-generator generate -i openapi/dcos.yaml -g go -o dcos --skip-validate-spec -D packageName=dcos,withGoCodegenComment=true -t templates
```

Comment:
We shouldn't generate all the files, they're not necessary IMO. I had to specify only the files I wanted to generate in separate runs of `generate` command - see https://github.com/dcos/client-go/pull/13/files#diff-06f2134279436229c113632db3603d73R30 for an example. This means we don't have the travis file generated, etc. which is just cruft for the repo.